### DISCUSSION TITLE
Suggestions for customer feedback UI

### DISCUSSION THREAD

**Discussion by slouie2** [twilio-labs/paste#1461] [Q&A] [answered]
Hi Paste team,

Does Paste have a pattern to record negative/positive customer feedback? My team is building out this page which returns the top 3 search results of the customer's input question. We'd like to design it in a way that will encourage the customer to give us feedback on each of the articles they've viewed ("this article solved my issue" or "this article didn't solve my issue").  
Ideally there should be an option for customers to somehow provide a positive feedback and a negative one (the ✔️ and **Ⅹ** are placeholders). I've looked at the Paste docs but couldn't find anything pertaining to a UI pattern relative to this.

![Screen Shot 2021-05-17 at 6 15 17 PM](https://user-images.githubusercontent.com/74426389/118567687-dafe8980-b73b-11eb-9dad-199d3445d05c.png)

**Comment by SiTaggart** [comment, marked as answer]
Hi @slouie2 

Thanks for your questions.

Before we get too far we're actually really interested in hearing some more information around this point

> We've decided against using generic patterns like thumbs up/down, smilies, "yes/no"

Specifically, why? What led to that decision? What were the alternatives explored? This will help us give some feedback on preferred approach.

Generally speaking though, if you are primarily looking to increase engagement and get feedback from customers we believe that a simple "yes" "no" button set, Yes being the primary action and No being the secondary, would perfectly fit the goal and help bring some visual hierarchy to the page.

**Reply by slouie2** [reply]
Hi @SiTaggart, thanks for your reply! After some thought, we will go with a "yes/no" button set. I'll open a separate request for thumbs up/down icons. Feel free to close this discussion.
